include/emp/Evolve/World.hpp

```diff
@@ -1342,6 +1386,20 @@ namespace emp {
     return valid_IDs;
   }
 
+  /// Return IDs of all occupied neighbors of specified position.
+  template<typename ORG>
+  emp::vector<size_t> World<ORG>::GetValidNeighborOrgIDs(size_t id) { 
+    //Note: this function is similar to FindCellIDs, but because ORG don't know their index, 
+    //FindCellIDs can't be used.
+    emp::vector<size_t> valid_IDs(0);
+    for(size_t i = 0; i < pop.size(); i++) {
+      if ((bool) (pop[i].Raw()) && IsNeighbor(id, i)) {
```

Comment:
Would checking `IsOccupied` be equivalent to checking `pop[i].Raw()`?

Reply:
Quite possibly, this is modeled off of FindCellIDs